src/pages/api/signup.js

```diff
@@ -0,0 +1,55 @@
+export default function handler(req, res) {
```

Comment:
when this function will be used bro?

Reply:
It would be used as a dummy api when hitting endpoint 'localhost:3000/api/signup' on the sign up page, but not so sure is it the correct solution

Reply:
Ohh I see, didn't realize that the next js could do that. Yup for the dummy api handling it looks good to me 👍